inference-engine/src/vpu/graph_transformer/src/stages/strided_slice.cpp

```diff
@@ -39,14 +39,16 @@ class StridedSliceStage final : public StageNode {
         IE_ASSERT(numOutputs() == 1);
 
         std::vector<EnumSet<DataType>> expectedInputs3Types =
-            { {DataType::FP16}, {DataType::S32}, {DataType::S32} };
+            { {DataType::FP16, DataType::S32}, {DataType::S32}, {DataType::S32} };
         std::vector<EnumSet<DataType>> expectedInputs4Types =
-            { {DataType::FP16}, {DataType::S32}, {DataType::S32}, {DataType::S32} };
+            { {DataType::FP16, DataType::S32}, {DataType::S32}, {DataType::S32}, {DataType::S32} };
+
+        const auto outType = input(0)->desc().type();
 
         assertInputsOutputsTypes(
             this,
             numInputs() == 3 ? expectedInputs3Types : expectedInputs4Types,
-            {{DataType::FP16}});
+            {{outType}});
```

Comment:
Do we expect that first input precision and output precision are the same without bounding to FP16 or S32?